src/test/Components/CreateImageWizardV2/CreateImageWizard.content.test.tsx

```diff
@@ -475,36 +475,9 @@ describe('Step Custom repositories', () => {
       })
     );
 
-    await screen.findByText(/select all \(1016 items\)/i);
+    await screen.findByText(/select page \(10 items\)/i);
   });
 
-  test('filter works', async () => {
```

Comment:
I don't think we should be removing tests that don't pass even if it's after a big refactor. We'll still want to test if filtering repositories works.

When manually testing the filtering works well on real life data so in this case it's the MSW handlers/fixtures that will need to be updated for the mocked values to pass the tests as well. Since this might be a bigger task it should be okay to comment out the failing tests and split the fixtures updates to a separate PR